modules/reference/pages/rpk/rpk-acl/rpk-acl-delete.adoc

```diff
@@ -41,13 +41,17 @@ rpk acl delete [flags]
 |--allow-principal |strings |Allowed principal ACLs to remove
 (repeatable).
 
+|--allow-role |strings |Allowed role ACLs to remove (repeatable).
```

Comment:
```suggestion
|--allow-role |strings |Allowed role for ACLs to remove (repeatable).
```

Reply:
This flag is in the `rpk acl delete`, this command uses the flag to match the given ACL so we can delete. For example:

We create 2 ACLs for the same role (red-role), 1 that allows access to topic foo, 1 that deny access to topic bar:
```
$ rpk acl create --topic foo --operation all --allow-role red-role
$ rpk acl create --topic bar --operation all --deny-role red-role
```

If we want to remove the first, we should do:

```
$ rpk acl delete --topic foo --operation all --allow-role red-role
```

So maybe if the original wording is not clear we can change it to (same comment for `acl delete`):

```suggestion
|--allow-role |strings |Allowed role to remove this ACL from (repeatable).
```

What do you think? I plan to translate the feedback here to rpk so Paulo doesn't have to change that every release manually.

Reply:
Thanks @r-vasquez ! Much appreciated!

Reply:
Added this example into the text for rpk acl delete